### PR TITLE
change: [M3-7546] - Revert FW table row changes

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
Originally only the first three (3) services were listed on the table and any `X` additional services were included in the string `"plus X more."`

The new mock ups asked to list as many services possible with three (3) lines of text and if there were any additional services, then they would be denoted with an ellipsis `...`.

This PR reverts the changes on the Firewalls table rows back to the original implementation and removes the `clamp-js` dependency.

## Changes  🔄
List any change relevant to the reviewer.
- Removal of the `clamp-js` package.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| prev | neww |

## How to test 🧪
- Run the unit tests
- Visually inspect the UI for any defects

### Prerequisites
- Have a Firewall with more than 3 services (Linodes or NodeBalancers) assigned to it.

### Verification steps 
- Verify that the table row displays the first 3 services and any additional services are denoted in the "plus X more." string.

## As an Author I have considered 🤔
- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.tsx [hunk 1]
```diff
@@ -145,29 +113,30 @@ const getDevicesCellString = (
...
-        ))}
-      </div>
-    </StyledDivWrapper>
+    <>
+      {firstThree.map((thisDevice, idx) => (
+        <Link
+          className="link secondaryLink"
+          data-testid="firewall-row-link"
+          key={thisDevice.id}
+          to={`/linodes/${thisDevice.entity.id}`}
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.tsx:129]: We need NodeBalancers to link to `/nodebalancers/:id`. Currently, NodeBalancers are getting linked to `/linodes/:id`

**Review by bnussman-akamai — CHANGES_REQUESTED** [review]

File: packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.tsx (hunk 1 above)
Comment (tyler-akamai) [reply packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.tsx:129]:
```
to={`/${thisDevice.entity.type}s/${thisDevice.entity.id}`}
```

**Review by bnussman-akamai — APPROVED** [review]
Looks good pending all relevant automation passes!

The changeset isn't super crutial because this change was never shipped.

**Review by dwiley-akamai — APPROVED** [review]
Three services + "plus X more" displays when a firewall has >3 services assigned to it ✅
Unit test passes ✅
Code review ✅

**Comment by tyler-akamai** [comment]
LGTM 🚀

**Comment by bnussman-akamai** [comment]
There's a handful of e2e failures, but they don't look relevant to this work. Should be good to go. (might be related to https://status.linode.com/incidents/bw8zm1r1v992)

**Pull request merged**